### PR TITLE
Added Hyperparameter Challenge of Titanic and MNIST_CNN 

### PR DESCRIPTION
**Pull request by omkarghugarkar007**
The accuracy of the test set is 87.45% and the prediction file added. 
Hyper-Parameters tuned
1. Layer 1 - 64 nodes
2. Layer 2 -32 nodes
3.Layer 3 -32 nodes
4. Optimizer - Adam
5. Learning rate - 0.01
6. epochs - 50
7. Batch Normalization added
#10

### CODE REVIEW

**Comment by omkarghugarkar007** [comment]
Worked on the MNIST_CNN and got an accuracy of 98.25%
1. Added 3 Conv2D layers of filters 32,64 and 128
2. Added 128 nodes to the dense layer
3. Created a validation split of  0.2
#1

**Comment by dev9318** [comment]
Change the name of the file to TASK(no. of the task)_(your initials).

**Comment by omkarghugarkar007** [comment]
Yes, I have changed the names now. Thanks!

**Review by Aakriti28 — CHANGES_REQUESTED** [review]
Hey, do not commit your .ipynb checkpoints or changes to other files that do not involve your task done. Just take a look at the files changed once. Rest looks good!

**Comment by omkarghugarkar007** [comment]
Yeah done, I deleted the checkpoints. Rest files are good, only 2 tasks which I have completed, those files are changed. Thanks!

**Comment by omkarghugarkar007** [comment]
Is there any further change required and then it will be merged?
